modules/swagger-codegen/src/main/resources/LoopBackSwift/model.mustache

```diff
@@ -0,0 +1,27 @@
+{{#models}}{{#model}}import ObjectMapper
+import Realm
+
+public class {{classname}}: Object, Mappable, CustomStringConvertible {
+{{#vars}}{{#isEnum}}
+    public enum {{datatypeWithEnum}}: String { {{#allowableValues}}{{#values}}
+        case {{enum}} = "{{raw}}"{{/values}}{{/allowableValues}}
+    }
+{{/isEnum}}{{/vars}}
+{{#vars}}{{#isEnum}}    public dynamic var {{name}}: {{{datatypeWithEnum}}}?{{#defaultValue}} = {{{defaultValue}}}{{/defaultValue}}{{/isEnum}}{{^isEnum}}    public dynamic var {{name}}: {{datatype}}?{{#defaultValue}} = {{{defaultValue}}}{{/defaultValue}}{{/isEnum}}
+{{/vars}}
+
+    public var description: String {
+        return "{{classname}}({{#vars}}\n    {{name}}: \({{name}}){{#hasMore}},{{/hasMore}}{{^hasMore}}\n{{/hasMore}}{{/vars}})"
+    }
+
+    public required convenience init?(_ map: Map) {
+        self.init()
+    }
+
+    public func mapping(map: Map) {
+        {{#vars}}{{name}} <- map["{{name}}"]{{#hasMore}}
```

Comment:
May possibly cause a build failure if the `<-` function isn't defined for types of both operands.